### DISCUSSION TITLE
Creole horizontal line color different from border?

### DISCUSSION THREAD

**Discussion by Potherca** [plantuml/plantuml#1480] [Q&A]
[Creole horizontal lines](https://plantuml.com/creole#acc8003baf7c883f) are colored the same color as the borders of the object they are in.

Is it possible to color the horizontal line in another color than that of the border?

To demonstrate what I mean:

[![](http://cdn-0.plantuml.com/plantuml/png/LOyn3i8m34NtdCBgpXceO84xT74RIur8xSes2uYxawHI17-otl_sbqTpL7zkETWZyOwA6ooeyI8QIQ-IHK4fXb0X33VHz19sp90Jam4gJNmgcdYfOOhWAu7XrZbQGtExQ0Ey0nInEBMXwcwG77OnItCcS85shFjrV_slXiJTHfQIq6EfXB5Rlytd1edZqVq9OvdBvry0)](http://www.plantuml.com/plantuml/uml/LO-nJiOm38HtFyMfEyz01GkyGnUtiTI8rAxii81uTz8KHVotvRlpdJ97PQ-VHo6AZwmdEn_OsDEBUH9_jM8EbqHqGKnlvldBj791oIe1TbdqgMdHxGfBGjq5mSVW71sjlJ4cV1EQrAeG_lGUo1MdHUIr2AgXbmt_kjyVfv5rsApM4duVjJ0Fw__oSuUYwTx-8PhRe_tz5m00)

I would like the lines between the words to be another color then red. How can I achieve this?

<sub>This question on the forum: https://forum.plantuml.net/18036/creole-horizontal-line-color-different-from-border</sub>

**Comment by The-Lum** [comment]
Hi @Potherca, and all,

>Is it possible to color the horizontal line in another color than that of the border?

It seems that is not yet possible.

---
@PlantUML team, @arnaudroques:
Then, _to manage that_, perhaps add a new [`SName`](https://github.com/plantuml/plantuml/blob/master/src/net/sourceforge/plantuml/style/SName.java) like:
- `line`
- or `horizontalLine`

And a proposal to use on style like:
```puml
<style>
line {
  LineColor green
  LineThickness 1.5
  LineStyle 3-3
}
</style>
```

---

Regards,
Th.
